packages/docs/developer-resources/contractkit/data-encryption-key.md

```diff
@@ -0,0 +1,24 @@
+# Data Encryption Key
+
+An account may register a data encryption key (DEK) that can be used for lightweight signing or encryption operations. Some examples of DEK usage are:
```

Comment:
would be great to add DEK and ODIS to the glossary too

Reply:
+1 great idea!